UnityProject/Assets/Scripts/Actors/PlayerController.cs

```diff
@@ -55,11 +59,22 @@ private void Move()
             animator.SetFloat("speed_front",actualSpeed*_inputManager.Vertical);
             animator.SetFloat("speed_right",actualSpeed*_inputManager.Horizontal);
             _movementDirection = new Vector2(_inputManager.Horizontal, _inputManager.Vertical);
-            _velocity = _movementDirection * actualSpeed;
-            rb.velocity = _velocity;
             
-            if (_inputManager.Dash)
+            //If character is moving diagonally, multiply the speed vector by 0.7. Else the character will move at the normal speed * sqrt 2
+            if (_movementDirection.x != 0 && _movementDirection.y != 0)
```

Comment:
This should only be applied if the player is using a keyboard.
Controllers apply this 70% diagonal correction automatically so you don't want to double it.